### PR TITLE
Bug/securetoken

### PR DESCRIPTION
**Pull request by nachiket87**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fixed issue [#39  ]

## Added to documentation?

- [ ] readme
- [X] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text
Image of Fix

### CODE REVIEW

**Comment by nachiket87** [comment]
Are there any reviewers?

**Comment by clairefro** [comment]
> Are there any reviewers?

Hi! Thanks again for this feature fix. 

We haven't forgotten this and will get to it soon. Apologies that we've been internally backlogged and it's taken us some time to review.

File: commands/config.js [hunk 1]
```diff
@@ -15,7 +16,7 @@ export const config = (cmdObj) => {
     console.log(chalk.cyanBright("Your current config:"));
     console.log(`github username:  ${getConfig("github")}`);
     console.log(
-      `token:            ${getConfig("token") ? "<hidden>" : "undefined"}`
+      `token:            ${getConfig("token") ? `<hidden>` : "undefined"}`
```
Comment (KmBarnett) [inline commands/config.js:19]: Is there a reason quotes had to be changed? For consistency purposes, if they didn't need to be changed, revert this back.

File: program.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ export const createProgram = (args) => {
     .description("Set up or view config (Github credentials etc.)")
     .option("-v, --view", "view current config")
     .option("-g, --github <username>", "set github username")
-    .option("-t, --token <token>", "set github personal access token")
+    .option("-t, --token", "set github personal access token")
```
Comment (KmBarnett) [inline program.js:47]: `<token>` made the token a required argument for this command; Why did you remove it? Setting a Github token to undefined and getting the message `Successfully set GitHub personal access token.` is a bad flow.

File: commands/new.js [hunk 3]
```diff
@@ -18,6 +18,14 @@ const ghUsernameRegex = new RegExp(
   "i"
 );
 
+const TOKEN = {
```
Comment (KmBarnett) [inline commands/new.js:21]: This file is for the creation of new projects not for adding to the config. Lines 21-28 and lines 112-119 need to be extracted to a new file

**Review by KmBarnett — CHANGES_REQUESTED** [review]
The code is mostly good it just needs to be restructured. There are a few that need to be undone or explained.

**Comment by nachiket87** [comment]
Accidentally request review again. Please ignore.

**Comment by nachiket87** [comment]
I currently have an unsolvable node / yarn issue in running the cli on dev. I don't think I can fix this. Thanks for the time.

**Pull request closed without merging**